scipy/special/basic.py

```diff
@@ -2132,7 +2132,7 @@ def comb(N, k, exact=False, repetition=False):
 
     Returns
     -------
-    val : int, float, ndarray
+    val : float or an int(depending on the value of `exact`), ndarray
```

Comment:
This is too long as a type descriptor. I would propose ``scalar or ndarray``, and then in the sentence below explain when you get what.